.eslintignore

```diff
@@ -8,8 +8,6 @@ packages/*
 # List of packages to Lint
 !packages/medusa
 
-integration-tests/*
```

Comment:
@srindom @olivermrbl in order to be able to lint and cleanup the tests I was working on, I had to remove these ignore. wdyt?